### DISCUSSION TITLE
如何更改日志的输出等级？

### DISCUSSION THREAD

**Discussion by fatinghenji** [nonebot/discussions#54] [问答]
在使用插件的时候出现问题，在向插件开发者反馈时，要求调低日志等级后再提再供相关日志文件。
翻阅了下[指南](https://v2.nonebot.dev/guide/)，没有看到相关的说明……

**Comment by mnixry** [comment]
NoneBot1还是2？要调到什么等级？

**Reply by fatinghenji** [reply]
在使用的是nonebot2，应该是尽量的低？（我使用的是默认的日志输出）

**Reply by jks15satoshi** [reply]
> 在使用的是nonebot2，应该是尽量的低？（我使用的是默认的日志输出）

NoneBot 2 默认的 logger 可以通过修改 [`log_level`](https://v2.nonebot.dev/api/config.html#log-level) 配置调整（如何修改配置参见 [基本配置](https://v2.nonebot.dev/guide/basic-configuration.html) 部分）。自定义 logger 参见 [loguru 文档](https://loguru.readthedocs.io/en/stable/api/logger.html#loguru._logger.Logger.add)。
